tests/test_integration.py

```diff
@@ -13,29 +13,29 @@ def client():
     with test_app.test_client() as client:
         yield client
 
-@patch('requests.get')
-def test_index_page(mock_get_requests, client):
-    mock_get_requests.side_effect = mock_get_lists
 
-    response = client.get('/')
-    response_html = response.data.decode()
+def mock_get_items(collection):
+    items = [
+        Item(1, 'Done', 'Task 1', date.today()),
+        Item(2, 'Doing', 'Task 2', date.today()),
+        Item(3, 'ToDo', 'Task 3', date.today()),
+    ]
+    return items
 
-    assert "Task 1" in response_html
+def mock_get_db_collection():
+    return
 
-def mock_get_lists(url, params):
-    if url == f'https://api.trello.com/1/boards/{os.getenv("BOARD_ID")}/cards':
-        response = Mock(ok=True)
-        response.json.return_value = get_sample_trello_cards_response()
-        return response
+def test_index_page(monkeypatch, client):
+    monkeypatch.setattr(
```

Comment:
You've done the hard part here of patching; the only reason this test still seems to be failing is because there's no relevant environment variables set in your `.env.test` file. Given that the values don't matter, we could equally load them in explicitly during the test, e.g. in the client function:
```
    os.environ['MONGO_DB_CONNECTION']='invalid'
    os.environ['MONGO_DB_DATABASE']='test'
```